CHANGES

```diff
@@ -33,12 +34,18 @@ Notable User Facing Changes
   kernel segfaults.
 - Added a new POCL_CPU_LOCAL_MEM_SIZE environment for overriding the
   local memory size for the CPU devices.
+- Partial support for cl_khr_fp16 when compiled with LLVM 16+. The
+  implementation relies on Clang, and may result in emulation if
+  the CPU does not support the required instruction set.
 
 Notable Fixes
 -------------
 
 - Fix handling of clBuildProgram() -I directories given with quotes or
   with leading/trailing white space.
+- CPU devices now flush printf() output after each call instead of waiting
```

Comment:
Why did you move this to fixes section? I think it could be considered a feature strictly put.

Reply:
fixed